apps/crawler-fetcher/tests/python/test_podcast_resolve_url.py

```diff
@@ -34,21 +34,21 @@ def test_get_feed_url_from_google_podcasts_url():
     assert _get_feed_url_from_google_podcasts_url('http://www.example.com/') == 'http://www.example.com/'
     assert _get_feed_url_from_google_podcasts_url('totally not an URL') == 'totally not an URL'
 
-    npr_feed_url = 'https://feeds.npr.org/381444908/podcast.xml'
```

Comment:
this feed url seems to oscillate at times for no reason, so hopefully the FT one will be more stable